invenio_rdm_records/records/dumpers/pids.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""DataCite-based data model for Invenio."""
+
+
+from invenio_records.dumpers import ElasticsearchDumperExt
+
+
+class ExternalPIDsDumperExt(ElasticsearchDumperExt):
+    """Elasticsearch dumper extension for External PIDs support.
+
+    The JSON representation of the external PIDs is a dictionary which keys
+    are the identifier's scheme. However, that would turn into a dynamic
+    mapping in ES with `text` type attributes. However, we want to be able to
```

Comment:
super minor: double `However`
```suggestion
    mapping in ES with `text` type attributes. We want to be able to
```